 * Serving Flask app "bot" (lazy loading)
 * Environment: production
   WARNING: This is a development server. Do not use it in a production deployment.
   Use a production WSGI server instead.
 * Debug mode: on
Traceback (most recent call last):
  File "/home/ec2-user/saneora/saneora/viber_bot/__init__.py", line 10, in <module>
    start_bot(config.VIBER_AUTH_TOKEN, config.TMDB_API_KEY)
  File "/home/ec2-user/saneora/saneora/viber_bot/bot.py", line 415, in start_bot
    app.run(debug=True)
  File "/home/ec2-user/saneora/saneora_env/lib/python3.9/site-packages/flask/app.py", line 990, in run
    run_simple(host, port, self, **options)
  File "/home/ec2-user/saneora/saneora_env/lib/python3.9/site-packages/werkzeug/serving.py", line 984, in run_simple
    s.bind(server_address)
OSError: [Errno 98] Address already in use
